### PR TITLE
[bugfix] Fix rendering of reviews

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by ChrisBr** [comment]
@adrianschroeter @hennevogel please have a look

**Comment by adrianschroeter** [comment]
looks fine to me. (it is questionable that we set reviewer by default, but given that the api is doing it as well ...)

File: src/api/app/models/bs_request.rb [hunk 1]
```diff
@@ -590,6 +590,7 @@ def _assignreview_update_reviews(reviewer, opts, new_review = nil)
         review_comment = ""
         history_class = HistoryElement::ReviewAccepted
       end
+      review.reviewer = User.current.login
```
Comment (hennevogel) [inline src/api/app/models/bs_request.rb]: If there is no `User.current` this will barf...

**Comment by ChrisBr** [comment]
Monkeypatched only on IBS

**Pull request merged**